src/main/features/response/routes/how-much-owed.ts

```diff
@@ -43,7 +41,7 @@ export default express.Router()
       const claim: Claim = user.claim
 
       if (form.model.amount > claim.claimData.amount.totalAmount()) {
-        let totalAmount: string = NumberFormatter.formatMoney(claim.claimData.amount.totalAmount())
+        let totalAmount: string = NumberFormatter.formatMoney(claim.totalAmountTillToday)
         let error = new ValidationError()
         error.property = 'amount'
         error.constraints = { 'amount': 'Enter a valid amount between £1 and ' + totalAmount }
```

Comment:
`'amount'` doesn't need to be wrapped by `''`

Reply:
Done